README.md

```diff
@@ -1,10 +1,10 @@
 # Cloudflare Tunnel client
 
 Contains the command-line client for Cloudflare Tunnel, a tunneling daemon that proxies traffic from the Cloudflare network to your origins.
-This daemon sits between Cloudflare network and your origin (e.g. a webserver). Cloudflare attracts client requests and sends them to you
-via this daemon, without requiring you to poke holes on your firewall --- your origin can remain as closed as possible.
+This daemon sits between Cloudflared network and your origin (e.g. a webserver). Cloudflare attracts client requests and sends them to you
```

Comment:
"Cloudflare" should be more precise than Cloudflared
As Cloudflared is just a daemon connect to Cloudflare network. Cloudflared is not a network

Reply:
Hi @longz3r, by `Cloudflare network`, we mean cloudflared is between Cloudflare's edge network and your origin. We would prefer to keep the documentation this way. The rest of your changes looks good though. We are happy to accept this PR if you can revert this line.